.travis.yml

```diff
@@ -0,0 +1,18 @@
+language: node_js
```

Comment:
You are adding the travis files, but I have the impression that CI doesn't work yet, so why are you adding these files?